streams/runners/storm/metrics/src/main/java/com/hortonworks/streamline/streams/metrics/storm/ambari/AmbariMetricsServiceWithStormQuerier.java

```diff
@@ -62,6 +65,12 @@
             "__process-latency", "__execute-count", "__execute-latency"
     ).build();
 
+    private static final List<Pair<String, String>> METRICS_APPLY_WEIGHTED_AVERAGE_PAIR =
```

Comment:
any reason why this is not a map ?

Reply:
My bad. No need to be List of Pair. Will fix.